exercises/sum-of-multiples/sum_of_multiples.py

```diff
@@ -1,2 +1,2 @@
-def sum_of_multiples(limit, multiples):
+def sum(factors, limit):
```

Comment:
Note the order of arguments had to be switched.
In addition, we might want to include `old_sum = sum` in the stub, or something like that to help the student with the general confusion of reusing `sum`

Reply:
We should _not_ shadow builtin Python functions like **sum** in any change we make. If the name in the canonical data conflicts with a Python keyword or builtin the Python name must be retained.